tests/integration/api_v2/http/pendingTransactions.test.js

```diff
@@ -0,0 +1,164 @@
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			expect(response.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(response.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('returns transactions with known sender address', async () => {
+			const response = await api.get(`${endpoint}?includePending=true&senderAddress=${refTransaction.sender.address}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			expect(response.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(response.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('returns transactions with known sender publicKey', async () => {
+			const response = await api.get(`${endpoint}?includePending=true&senderPublicKey=${refTransaction.sender.publicKey}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			expect(response.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(response.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('returns transactions with known sender username', async () => {
+			const response = await api.get(`${endpoint}?includePending=true&senderUsername=${refTransaction.sender.username}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			expect(response.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(response.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('empty transaction transactionId ->  ok', async () => {
+			const response = await api.get(`${endpoint}?includePending=true&transactionId=`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			expect(response.data[0]).toMap(pendingTransactionSchemaVersion5);
+			expect(response.data[0]).toEqual(
+				expect.objectContaining({
+					isPending: true,
+				}),
+			);
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('empty transaction moduleAssetId ->  ok', async () => {
+			const response = await api.get(`${endpoint}?includePending=true&moduleAssetId=`);
```

Comment:
Refer to the comments on the corresponding testcases in `tests/integration/api_v2/rpc/pendingTransactions.test.js`.